File: problems/python/completos/1012.py
PI = 3.14159
linha = input().split()
A = float(linha[0])
B = float(linha[1])
C = float(linha[2])

triangulo = (A * C) / 2
circulo = PI * C ** 2
trapezio = (((A + B) * C) / 2)
quadrado = B ** 2
retangulo = A * B

print("TRIANGULO: %.3f\nCIRCULO: %.3f\nTRAPEZIO: %.3f\nQUADRADO: %.3f\nRETANGULO: %.3f" % (triangulo, circulo, trapezio,
                                                                                           quadrado, retangulo))

File: ruby/complete/1012.rb
A, B, C = gets.chomp.split(" ").map(&:to_f)
PI = 3.14159

TRIANGULO = (A * C) / 2
CIRCULO = PI * C * C
TRAPEZIO = ((A + B) * C) / 2
QUADRADO = B * B
RETANGULO = A * B


puts format("TRIANGULO: %.3f\nCIRCULO: %.3f\nTRAPEZIO: %.3f\nQUADRADO: %.3f\nRETANGULO: %.3f",
TRIANGULO, CIRCULO, TRAPEZIO, QUADRADO, RETANGULO)
